tests/drivers/spi/spi_loopback/src/spi.c

```diff
@@ -58,7 +58,14 @@ static struct spi_dt_spec spi_slow = SPI_DT_SPEC_GET(SPI_SLOW_DEV, SPI_OP(FRAME_
 #define STACK_SIZE (512 + CONFIG_TEST_EXTRA_STACK_SIZE)
 #define BUF_SIZE 18
 #define BUF2_SIZE 36
+
+#ifdef CONFIG_DSPI_MCUX_EDMA
+/*DSPI DMA need aligned buffer for internal*/
+#define BUF3_SIZE 1440
```

Comment:
1440 or 2440 (see your commit message) ?

Reply:
1440 typo

Reply:
ok. Btw in your commit message: s/fixing:/Fixes: